Automattic/wp-calypso
# Analytics: Remove problematic "blacklist"/"whitelist" language #43395

#### Changes proposed in this Pull Request

* Rename `isUrlBlacklistedForPerformance` to `isUrlExcludedForPerformance`

#### Testing instructions

* Ensure unit tests pass
* Turn off your browser's "Do Not Track" option, enable `ad-tracking` config, and give GDPR consent. Then navigate to a page that is excluded from analytics (`/log-in`) and verify that analytics are blocked by checking `isAdTrackingAllowed` in the browser console and verifying it is `false`.

Addresses part of https://github.com/Automattic/wp-calypso/issues/43395
